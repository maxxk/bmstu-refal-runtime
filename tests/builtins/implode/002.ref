$ENTRY Go
{
	= <Prout <Lenw<Implode 'UTqwa!sdd3' 23>>>;
}