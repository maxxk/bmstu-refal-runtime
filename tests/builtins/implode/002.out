7 UTqwa !sdd323 
